sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryScope.kt

```diff
@@ -0,0 +1,114 @@
+package io.sentry.kotlin.multiplatform
+
+import io.sentry.ScopeCallback
+import io.sentry.kotlin.multiplatform.extensions.*
+
+actual class SentryScope : ISentryScope {
+
+    // We are directly modifying the Android SDK scope
+    private var scope: AndroidSentryScope? = null
+
+    actual override var level: SentryLevel?
+        set(value) {
+            scope?.level = value?.toAndroidSentryLevel()
+        }
+        get() {
+            return scope?.level?.toKMPSentryLevel()
+        }
+
+    actual override var user: SentryUser?
+        set(value) {
+            value?.onFieldChanged = {
+                scope?.user = value?.toAndroidSentryUser()
+            }
+        }
+        get() {
+            return scope?.user?.toKMPSentryUser()
+        }
+
+    /**
+     * Initializes this KMP Scope with the Android Scope
+     */
+    internal fun initWithAndroidScope(androidScope: AndroidSentryScope) {
+        this.scope = androidScope
+    }
```

Comment:
Should this be a ctor?

Reply:
It can be, but the KMP Scope creation has to be put inside the `configureScope` callback so we can access the Platform Scope. Shouldn't be a problem though I think